ONSdigital/sdc-responses-dashboard
# RAS-982 high security vulnerabilities

# What and why?
This PR addresses the vulnerabilities in the repo. It isn't ideal as we are still using gulp (which hasn't been updated for 5 years) and I have had to override its use of glob-parent as well as chokidar. I have remove JS linting and prettier, of courses it would be better to keep them, but they currently don't work and when I attempted to fix them there is all sorts of issues with dependencies. Considering they are both JS aimed which we have no intention of changing any time soon the impact is limited

# How to test?
Go to http://localhost:8078/dashboard after deploying this PR and make sure it works
